test/integration/crcsuite/util.go

```diff
@@ -22,14 +23,23 @@ func DeleteCRC() error {
 // Remove CRC home folder ~/.crc
 func RemoveCRCHome() error {
 
-	err := os.RemoveAll(CRCHome)
+	keepFile := filepath.Join(CRCHome, ".keep")
 
-	if err != nil {
-		fmt.Printf("Problem deleting CRC home folder %s.\n", CRCHome)
+	_, err := os.Stat(keepFile)
+	if os.IsNotExist(err) {
```

Comment:
dunno if it's better to be explicit with err?
`if err != nil && os.IsNotExist(err) {`

Reply:
Could it be that you meant `if err != nil || os.IsNotExist(err) {`? To cover the situation that there is an error, but it's not the `IsNotExist` one. Correct?

Reply:
Ah, I was thinking of doing the opposite, if `~/.crc/.keep` is not there for sure, then delete `~/.crc`, otherwise keep it. From that perspective, what I suggested looks fine to me, "if there was an error, and this error indicates `~/.crc/.keep` is not present, then delete `~/.crc`"
Note that my suggestion is not different from the current `if os.IsNotExist(err) {`, as `if os.IsNotExist(nil) {` is going to be false.

If you prefer the other way, always delete unless you are sure .keep is present, then I don't think the current code does that.